### PR TITLE
Issue 453: Update the version for ahash

### LINKED ISSUE #453: Update the version for ahash
**Problem description**
Currently in Cargo.toml we are using the crate 'ahash=0.6.2' as dependency which has been yanked, causing the build to fail.
image

**Problem location**
CI build failures.

**Suggestions for an improvement**
Update the version for ahash to latest in Cargo.toml.

### PR DESCRIPTION
**Pull request by AnkurAnand11**
**Change log description**  
Update the ahash version to 0.8.6

**Purpose of the change**  
Fixes #453 

**What the code does**  
Update ahash verion to 0.8.6 

**How to verify it**  
All tests should pass

### CODE REVIEW

File: nodejs/src/stream_manager.rs [hunk 1]
```diff
@@ -544,7 +544,7 @@ impl StreamManager {
             scaling_policy.scaling.clone(),
             retention_policy.retention.clone(),
             match tags.len() {
-                l if l == 0 => None,
+                0 => None,
```
Comment (AnkurAnand11) [inline nodejs/src/stream_manager.rs:547]: Updated this line as per the suggestion provided in the failed build logs.
image

File: src/sync/synchronizer.rs [hunk 2]
```diff
@@ -268,7 +268,7 @@ impl Synchronizer {
                     key: inner,
                     key_version: version,
                 };
-                let inner_map = self.in_memory_map.entry(outer_key).or_insert_with(HashMap::new);
+                let inner_map = self.in_memory_map.entry(outer_key).or_default();
```
Comment (AnkurAnand11) [inline src/sync/synchronizer.rs:271]: Updated this line as per the suggestion provided in the failed build log.
image

File: src/sync/synchronizer.rs [hunk 3]
```diff
@@ -413,7 +413,7 @@ impl Update {
 
         self.insert.push(insert);
         // also insert into map.
-        let inner_map = self.map.entry(outer_key.clone()).or_insert_with(HashMap::new);
+        let inner_map = self.map.entry(outer_key.clone()).or_default();
         inner_map.insert(inner_key, Value { type_id, data });
```
Comment (AnkurAnand11) [inline src/sync/synchronizer.rs:417]: Updated this line as per the suggestion provided in the failed build log.
image

File: src/sync/synchronizer.rs [hunk 4]
```diff
@@ -872,7 +872,7 @@ fn apply_inserts_to_localmap(
             let in_mem_inner_map = table_synchronizer
                 .in_memory_map
                 .entry(update.outer_key.clone())
-                .or_insert_with(HashMap::new);
+                .or_default();
```
Comment (AnkurAnand11) [inline src/sync/synchronizer.rs:875]: Updated this line as per the suggestion provided in the failed build log.
image

File: src/sync/synchronizer.rs [hunk 5]
```diff
@@ -903,7 +903,7 @@ fn apply_deletes_to_localmap(to_delete: &mut Update, table_synchronizer: &mut Sy
         let in_mem_inner_map = table_synchronizer
             .in_memory_map
             .entry(delete.outer_key.clone())
-            .or_insert_with(HashMap::new);
+            .or_default();
```
Comment (AnkurAnand11) [inline src/sync/synchronizer.rs:906]: Updated this line as per the suggestion provided in the failed build log.
image

**Review by tkaitchuck — COMMENTED** [review]
Check to make sure there aren't any other toml files bringing in aHash.

**Comment by AnkurAnand11** [comment]
ahash dependency is being used in only one Cargo.toml file.

**Review by anishakj — APPROVED** [review]
LGTM

**Pull request merged**